lms/djangoapps/discussion/rest_api/urls.py

```diff
@@ -24,31 +24,31 @@
 ROUTER.register("comments", CommentViewSet, basename="comment")
 
 urlpatterns = [
-    url(
+    re_path(
         r"^v1/courses/{}/settings$".format(
             settings.COURSE_ID_PATTERN
         ),
         CourseDiscussionSettingsAPIView.as_view(),
         name="discussion_course_settings",
     ),
```

Comment:
`UploadFileView` is not imported in that change and that was cuasing error so I had remove it since that view might have been added after maple release branch is cut.